src/server/templates/register-form/summary.njk

```diff
@@ -36,7 +36,15 @@
         {{ row('NHS number', {'empty': 'I don’t know my NHS number'}, 'register-form:nhsNumber') }}
       {% endif %}
     </table>
-
+    <table class="summary">
+      <caption>Armed force</caption>
+		{% if data.previouslyArmed['previously-armed'] %}
```

Comment:
please change tabs to spaces with 2sp indent

does your editor support [editorconfig](http://editorconfig.org/)? if not can you configure it to match our rules?  https://github.com/nhsuk/register-with-a-gp-beta-web/blob/master/.editorconfig